### ISSUE TITLE
Potential early-adopter ✋, How to use it?

### ISSUE THREAD

**Issue by escorciav** [pytorch/nestedtensor#243]
Hi @fmassa!

I like your next `NestedTensor` abstraction.

- Should I take it [by cp-ing this](https://github.com/facebookresearch/detr/blob/d45a12c72fd9877f896aad566eee8478e49cac95/util/misc.py#L339:L359) into my codebase?

- Would you recommend to use [this package](https://github.com/pytorch/nestedtensor)? Can I pip install it?

_My use case_

- I'm currently passing a dictionary into a couple of modules (loss, layers). However, each module take different tensors.

- I did an awful hack in my dataloader to support this.

It seems that your abstraction serves my use case. That's what I understood from your DETR project. Am I wrong?

Cheers :beers:

**Comment by escorciav** [comment]
Closing the issue as I misunderstood the purpose. Yet, I will keep an :eye: on this package.
